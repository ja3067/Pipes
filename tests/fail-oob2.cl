l = [12.4, 37.6, 46.5, 1]

print(1) # should get to here

c = l[3]

print(2) 

d = l[12]

print(2) # shouldn't get to here

