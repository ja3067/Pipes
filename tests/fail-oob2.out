1
2
RuntimeError: list index out of bounds
